README.md

```diff
@@ -76,65 +83,156 @@ Quick Start
         }
     }
     ```
+    
```

Comment:
Maybe we could add steps here too like 

1. Integrate the web SDK PayPal or Venmo flow into your website

PayPal Example
...
Venmo Example
...

2. Include the Popup Bridge iOS SDK in your iOS app

... whatever code is needed to include the SDK

3. Set up the `POPPopupBridge` with your `WKWebview`:

`popupBridge = POPPopupBridge(webView: webView)`

4. Load your website containing the PayPal or Braintree Web SDK (Step 1) in your `WKWebView`

`webView.load(URLRequest(url: URL(string: "https://merchant-site.com")!))

Reply:
A missed that this already exists (only looking at the changes) - disregard